src/constants/route.js

```diff
@@ -12,6 +12,7 @@ const PATH = {
 
   // Full-screen
   HOME: '/home',
+  SIGN_IN: '/signin',
```

Comment:
I didn't see this when I resolved merge conflicts.  We need to delete line 8 that says `SIGN_IN: '/tabs/signin',`... my bad.